larray_editor/editor.py

```diff
@@ -919,155 +1025,32 @@ def load_example(self):
                 filepath = AVAILABLE_EXAMPLE_DATA[dataset_name]
                 self._open_file(filepath)
 
-    #########################################
-    #               HELP MENU               #
-    #########################################
-
-    def open_documentation(self):
-        QDesktopServices.openUrl(QUrl(get_documentation_url('doc_index')))
-
-    def open_tutorial(self):
-        QDesktopServices.openUrl(QUrl(get_documentation_url('doc_tutorial')))
-
-    def open_api_documentation(self):
-        QDesktopServices.openUrl(QUrl(get_documentation_url('doc_api')))
-
-    def report_issue(self, package):
-        def _report_issue(*args, **kwargs):
-            if PY2:
-                from urllib import quote
-            else:
-                from urllib.parse import quote
-
-            versions = get_versions(package)
-            issue_template = """\
-## Description
-**What steps will reproduce the problem?**
-1. 
-2. 
-3.
- 
-**What is the expected output? What do you see instead?**
-
-
-**Please provide any additional information below**
-
-
-## Version and main components
-* Python {python} on {system} {bitness:d}bits
-"""
-            issue_template += "* {package} {{{package}}}\n".format(package=package)
-            for dep in dependencies[package]:
-                issue_template += "* {dep} {{{dep}}}\n".format(dep=dep)
-            issue_template = issue_template.format(**versions)
-
-            url = QUrl(urls['new_issue_{}'.format(package)])
-            if PYQT5:
-                from qtpy.QtCore import QUrlQuery
-                query = QUrlQuery()
-                query.addQueryItem("body", quote(issue_template))
-                url.setQuery(query)
-            else:
-                url.addEncodedQueryItem("body", quote(issue_template))
-            QDesktopServices.openUrl(url)
-
-        return _report_issue
 
-    def open_users_group(self):
-        QDesktopServices.openUrl(QUrl(urls['users_group']))
-
-    def open_announce_group(self):
-        QDesktopServices.openUrl(QUrl(urls['announce_group']))
-
-    def about(self):
-        """About Editor"""
-        kwargs = get_versions('editor')
-        kwargs.update(urls)
-        message = """\
-<p><b>LArray Editor</b> {editor}
-<br>The Graphical User Interface for LArray
-<p>Licensed under the terms of the <a href="{GPL3}">GNU General Public License Version 3</a>.
-<p>Developed and maintained by the <a href="{fpb}">Federal Planning Bureau</a> (Belgium).
-<p>&nbsp;
-<p><b>Versions of underlying libraries</b>
-<ul>
-<li>Python {python} on {system} {bitness:d}bits</li>
-"""
-        for dep in dependencies['editor']:
-            if kwargs[dep] != 'N/A':
-                message += "<li>{dep} {{{dep}}}</li>\n".format(dep=dep)
-        message += "</ul>"
-        QMessageBox.about(self, _("About LArray Editor"), message.format(**kwargs))
+class ArrayEditor(AbstractEditableEditor):
```

Comment:
I fear having the file (does it even work?) and help menus will feel weird/out of place in the array editor, especially when called from within the viewer itself. The undo/redo functionality is useful though.